packages/compass-e2e-tests/helpers/buildinfo.ts

```diff
@@ -0,0 +1,76 @@
+import assert from 'node:assert/strict';
+
+// subsets of the hadron-build info result
+
+const commonKeys = ['productName'];
+type CommonBuildInfo = Record<typeof commonKeys[number], string>;
+
+function assertObjectHasKeys(obj: any, name: string, keys: readonly string[]) {
```

Comment:
I'd suggest almost always opt for `unknown` instead of `any`.

```suggestion
function assertObjectHasKeys(obj: unknown, name: string, keys: readonly string[]) {
```